test/compress_test.js

```diff
@@ -23,6 +23,8 @@ exports.compress = {
       actual.push(entry.path);
     });
     parse.on('close', function() {
+      actual.sort();
+      expected.sort();
```

Comment:
I just stumbled upon this and don't know very much about the code here. I looked briefly at the `unzip` and as far as I can see, it will emit file entries in the order it finds them in the zip file. That's what made me conclude that you do have a well-defined order for the paths in this test.

Maybe that's subject to change and if so, sorting the lists you get makes sense.